File: src/main/java/space/bielsolososdev/noto/api/model/user/UserResponse.java
package space.bielsolososdev.noto.api.model.user;

import java.time.LocalDateTime;
import java.util.Set;
import java.util.UUID;

public record UserResponse(
        UUID id,
        String username,
        String email,
        boolean isActive,
        LocalDateTime createdAt,
        Set<String> roles
) {}

File: src/main/kotlin/space/bielsolososdev/noto/api/model/user/UserResponse.kt
package space.bielsolososdev.noto.api.model.user

import java.time.LocalDateTime

data class UserResponse(
    val id: Long,
    val username: String,
    val email: String,
    val isActive: Boolean,
    val createdAt: LocalDateTime?,
    val roles: Set<String>
)

